.gitattributes

```diff
@@ -16,7 +17,6 @@
 *.root  -text -diff
 *.gz     -text -diff
 *.nii     -text -diff
-*.sh    eol=lf
```

Comment:
let's keep this. There's some scripts in examples, SimSET etc as well.

Please commit with `[ci skip]`